src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj

```diff
@@ -23,13 +24,13 @@
     <ProjectReference Include="$(NuGetCoreSrcDirectory)NuGet.Common\NuGet.Common.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == '$(NETFXTargetFramework)' or '$(TargetFramework)' == 'net45' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(NETFXTargetFramework)' or '$(TargetFramework)' == 'net45' or '$(TargetFramework)' == 'netstandard2.0' ">
     <Reference Include="System.Security" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
```

Comment:
Same as previous comment, doesn't the .NET SDK add these BCL references automatically for .NET Standard?

Reply:
The netstandard condition was removed from NuGet.Common.csproj, but not here.